src/containers/reports/CrisisReportView.js

```diff
@@ -204,28 +237,129 @@ class CrisisTemplateContainer extends React.Component<Props> {
     });
   }
 
+  handleEditClick = () => {
+    this.setState({
+      edit: true
+    });
+  }
+
+  handleShowDiscard = () => {
+    this.setState({
+      showDiscard: true
+    });
+  }
+
+  handleCloseDiscard = () => {
+    this.setState({
+      showDiscard: false
+    });
+  }
+
+  handleShowDelete = () => {
+    this.setState({
+      showDelete: true,
+    });
+  }
+
+  handleCloseDelete = () => {
+    this.setState({
+      showDelete: false,
+    });
+  }
+
+  handleDiscard = () => {
+    const { actions, history } = this.props;
+    actions.clearCrisisTemplate();
+    history.push(HOME_PATH);
+  }
+
+  handleDelete = () => {
+    console.log('Doing delete things');
```

Comment:
is this doing delete things?

Reply:
Delete is gonna come in another PR. I can remove the console.log for now tho